Kitodo/src/main/java/org/kitodo/services/file/FileService.java

```diff
@@ -564,44 +564,40 @@ public URI getProcessBaseUriForExistingProcess(Process process) {
      *            the process to get the sublocation for.
      * @param processSubType
      *            The subType.
-     * @param id
+     * @param folderName
      *            the id of the single object (e.g. image) if null, the root
      *            folder of the sublocation is returned
      * @return The URI of the requested location
      */
-    public URI getProcessSubTypeURI(Process process, ProcessSubType processSubType, String id) {
+    public URI getProcessSubTypeURI(Process process, ProcessSubType processSubType, String folderName) {
```

Comment:
it's not always a folder name. can be an imagename for example (as stated in the javadoc)